docuilib/src/components/ComponentPage.tsx

```diff
@@ -60,10 +60,12 @@ export default function ComponentPage({component}) {
 
   const buildTabs = () => {
     const tabs = component.docs?.tabs;
-
+    const api = component.props;
+    const tabsArray = api ? [...tabs, devTab] : tabs;
+    
     // TODO: align Tabs bottom border with TabItem's selected indication line
     if (tabs) {
-      return <Tabs className="main-tabs">{getTabItems([...tabs, devTab])}</Tabs>;
+      return <Tabs className="main-tabs">{getTabItems(tabsArray)}</Tabs>;
```

Comment:
If we remove Dev tab based on props, it won't render things like notes and tips, no? 
Which component/page has this issue (has no api)?

Reply:
That's why I based it on the props - that the most important feature that without it I don't think we need the tab. 
Can you think of a case where we don't have props but need to display the extend/modifiers/snippet?

Reply:
Not really, approving, worse case we'll change it